### PR TITLE
Transfer documentation from module to README

### PR DESCRIPTION
**Pull request by Silverdev2482**
I think this is mostly good but I'm not sure about the syntax of how I named options, I also wouldn't be suprised as if there are a few spelling errors. It still need proofreading by me too.

### CODE REVIEW

**Comment by Silverdev2482** [comment]
Will `servers.<name.files` Only place non existing files, or will it overwrite them? I would like to specify that in the README.

**Comment by Silverdev2482** [comment]
Looking at it I believe it will put the files into a .bak when modified, but will overwrite the previous.bak file, if I am wrong please correct me.

File: README.md [hunk 1]
```diff
@@ -278,3 +367,6 @@ This example takes an attrset of the IDs and hashes for Modrinth mods, fetches e
   };
 }
 ```
+
+#### `servers.<name>.files`
+Things to copy into this server's data directory. Similar to symlinks, but these are actual files. Useful for configuration files that don't behave well when read-only.
```
Comment (Infinidoge) [inline README.md:372]: `files` also has the option for generating files from data, which would be useful to document. CC @Misterio77 for advice on that.

**Review by Infinidoge — CHANGES_REQUESTED** [review]
Thanks for putting in the work of putting it into the file. I'm not too happy with how it is all laid out, but that is mostly my fault for how it is documented. Something I want to look into for the future would be generating the documentation like how it is done in Nixpkgs et al., as that would be the best solution.

Most of the suggestions are things that can just be committed as-is, let me know if you have any thoughts on them.

**Comment by Silverdev2482** [comment]
All suggestions look good to me, I wasn't too sure on format/syntax so I went to what seems the most similar with the existing readme. Not sure if I feel like documenting the files option, as I don't have the best understanding of the nix language. If @Misterio77 feels like documenting that we should wait, or maybe a make separate issue and pr. Other than that this seems all good to merge.

File: README.md [hunk 2]
```diff
@@ -231,17 +231,127 @@ This `fetchurl` invocation directly fetches the mod, and can be copy-pasted to w
...
+  "rcon.password" = "hunter2";
+}
+```
+
+#### `servers.<name>.package`
+
+The Minecraft server package to use. Example:
+`pkgs.minecraftServers.vanilla-1_18_2`
+
+#### `servers.<name>.package`
```
Comment (BatteredBunny) [inline README.md:346]: I'm assuming this is supposed to be ``jvmOpts`` instead?

**Review by BatteredBunny — COMMENTED** [review]
Noticed a typo

File: README.md (hunk 2 above)
Comment (Infinidoge) [reply README.md:346]: Surprised I didn't notice that, good catch.
```suggestion
#### `servers.<name>.jvmOpts`
```

**Comment by Silverdev2482** [comment]
Should that issue be created and this merged? Or do we add that to this PR and wait for it to be documented?

**Comment by Silverdev2482** [comment]
don't mean any disrespect, but should we just merge this now? If I am not mistaken there isn't anything I need to do, and Misterio hasn't responded. Every other suggestion looks fine and has been approved.

**Comment by Infinidoge** [comment]
Sorry for the wait, summer has been pretty busy for me so even though this is small and straight forward, it kept getting pushed back by the other things I've been doing.

**Pull request merged**